### PR TITLE
add new option and linked to new floor

### PR DESCRIPTION
**Pull request by caspero-62**
Add new "find a way out" option and link to new floor pages"

### CODE REVIEW

File: english/normal-dungeon/3/1.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+Great you found a way out!
+Are you sure you don't want to check out the dugeon?
+
+[Go back to front of dugeon](../0/0.md)
```
Comment (ming-tsai) [inline english/normal-dungeon/3/1.md:4]: nit: `Go back to front of dugeon` to `Go back to the front of the dungeon`

**Comment by caspero-62** [comment]
Hello @ming-tsai,
I just made the new change as requested.

**Comment by ming-tsai** [comment]
> Hello @ming-tsai,
> I just made the new change as requested.

Please make change `dugeon` to `dungeon`

**Comment by ming-tsai** [comment]
@caspero-62 welcome 🎉 and thanks for your contribution 🙌

**Pull request merged**